UberSignals.podspec

```diff
@@ -0,0 +1,19 @@
+Pod::Spec.new do |s|
+  s.name = 'UberSignals'
+  s.version = '1.0.0'
+  s.license = { :type => "Copyrighted", :file => "LICENSE.md" }
+  s.summary = 'Signals is an eventing framework that enables you to implement the Observable pattern without using NSNotifications.'
+  s.homepage = 'https://github.com/uber/signals-ios'
+  s.social_media_url = 'https://twitter.com/UberEng'
+  s.authors = { 'Tuomas Artman' => 'tuomas@uber.com' }
+  s.source = { :git => 'https://github.com/uber/signals-ios.git', :tag => s.version }
```

Comment:
Do we want to use v1.0.0 or 1.0.0 as the name for our tags?

If we want to use the former this should be `s.source = { :git => 'https://github.com/uber/signals-ios.git', :tag => "v#{s.version.to_s}" }